modules/core/src/main/scala/lucuma/odb/api/model/ObservationModel.scala

```diff
@@ -29,6 +30,9 @@ object ObservationModel extends ObservationOptics {
   implicit val TopLevelObservation: TopLevelModel[Observation.Id, ObservationModel] =
     TopLevelModel.instance(_.id, ObservationModel.existence)
 
+  implicit val EqObservation: Eq[ObservationModel] =
```

Comment:
I suppose we could have tests of these `Eq` instances

Reply:
Yeah you're right. It's temporary code but it may be around for a while and I've avoided it long enough I think.  I'll go back and add some arbitraries, etc.